README.md

```diff
@@ -1,101 +1,136 @@
+<div align="center">
 
-## Set-up GitHub Actions
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-Please do the following **steps in this order**:
+</div>
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+<!-- TABLE OF CONTENTS -->
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+# 📗 Table of Contents
 
-Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Todo List Project](#-todo-list-project)
+  - [🛠️ Built With ](#-built-with-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
 
-![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+<!-- PROJECT DESCRIPTION -->
 
-## Set-up linters in your local env
+# 📖 Todo List Project<a name="about-project"></a>
 
-**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
+**Minimalist To Do List Project**
+is a website that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. It is inspired by the minimalist website. That website is already offline but you can still play with thanks to web.archive.org.
 
-```
-# .gitignore
-node_modules/
-```
+## 🛠️ Built With <a name="built-with"></a>
 
-### [Lighthouse](https://developers.google.com/web/tools/lighthouse)
+    HTML
+    CSS
+    Javascript(ES6)
+    Webpack
 
-An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-You can get the Lighthouse report by any of the following ways:
+## 🚀 Live Demo <a name="live-demo" href='https://outkarm.github.io/To-Do-List-App/dist/'>Click Here to view</a>
```

Comment:
Great job @Outkarm 🤝

- [ ] [Optional] Kindly remove this live demo link as it is for a different project.

Reply:
@1GORDON  thanks for the tip, but I will put the demo link when i am done with the whole project